GiteaIntegrationTest.testGiteaMapping fails with latest Gitea

https://travis-ci.org/bozaro/git-as-svn/jobs/460845508

    java.lang.AssertionError: Expected SVNAuthenticationException to be thrown, but nothing was thrown
        at org.testng.Assert.expectThrows(Assert.java:1394)
        at org.testng.Assert.assertThrows(Assert.java:1364)
        at svnserver.ext.gitea.GiteaIntegrationTest.testGiteaMapping(GiteaIntegrationTest.java:257)

This didn't happen two days ago: https://travis-ci.org/bozaro/git-as-svn/builds/459794680

@zeripath

--- a/src/main/java/svnserver/ext/gitea/mapping/GiteaAccess.java
+++ b/src/main/java/svnserver/ext/gitea/mapping/GiteaAccess.java
@@ -75,6 +75,7 @@ final class GiteaAccess implements VcsAccess {
                   repository.getPermissions().setPush(false);
                   return repository;
                 }
+                throw new FileNotFoundException();
               } catch (ApiException e) {
                 if (e.getCode() == 404) {
                   throw new FileNotFoundException();

